validation/validator_test.go

```diff
@@ -98,7 +98,7 @@ func TestValidateParameterNotNil(t *testing.T) {
 	t.Run("Not nil value", func(t *testing.T) {
 		assert.Nil(t, NewParameterValidator().ValidateParameterNotNil("param", t).Status())
 	})
-	t.Run("Nil interface{}", func(t *testing.T) {
+	t.Run("Nil interface", func(t *testing.T) {
```

Comment:
No... it is really `interface{}`

Reply:
As discussed, this is a limitation of the sonar-scanner-cli tool. Without this fix I have:
```
11:04:35  ERROR: Error during SonarQube Scanner execution
11:04:35  java.util.regex.PatternSyntaxException: Illegal repetition near index 48
11:04:35  ^func\s+TestValidateParameterNotNil/Nil_interface{}\s*\(
```
As we both agreed, I've removed the incriminating '{}', and I've added a comment to explain my issue.